tests/config_server.py

```diff
@@ -3,7 +3,7 @@
 # This file is only needed for setting up a dedicated server configuration
 # with support for extracting username information and emailing users.
 
-SERVER_NAME = 'localhost'
+SERVER_NAME = 'localhost.localdomain'
```

Comment:
Why this change? I think this will lead to platform specific behaviour, no?

Reply:
@matthewwardrop so I noticed that the test runs have a bunch of [Flask warnings](https://travis-ci.org/airbnb/knowledge-repo/jobs/433841809#L1065-L1109), and this makes them go away. This config is only for running the test suite, right?

Reply:
@naoyak Yep... I was worried it might break Windows... but it didn't! Merging!